### PR TITLE
add unitest tests

### PR DESCRIPTION
**Pull request by jlecomte-IED**
https://github.com/fulll/datahub/issues/3649

Add unitest test for data engineer junior test

### CODE REVIEW

File: DataEngineer/Junior/test_bike_investigation.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        # TO DO : create a panda DataFrame from the data dictionary
+
+        result = time_stats(df)
+
+        self.assertEqual(result['mostCommonMonth'], ['january'])
+        # TO DO : add more tests for the other keys in the result dictionary
+
+    def test_time_stats_missing_data(self):
+        # TO DO : base on the above test, create tests for all function in bike_investigation.py and for couple of corner cases.
+
```
Comment (dangkhoa-dinh) [inline DataEngineer/Junior/test_bike_investigation.py:22]: We don't want candidate to test `get_filters` or `load_data` function IMHO.

```suggestion
    # TO DO : base on the above test, create tests for station_stats, trip_duration_stats and user_stats function. Make sure you cover common corner cases.
```

**Review by dangkhoa-dinh — CHANGES_REQUESTED** [review]
Small request change to clarify the requirement

**Review by shouze — APPROVED** [review]
